CONTRIBUTING.md

```diff
@@ -24,6 +24,66 @@ git checkout «your-branch»
 git rebase main
 ```
 
+## Testing helm charts locally
+
+First of all, you will need a kubernetes cluster available. For this, we suggest to use [KinD](https://kind.sigs.k8s.io/) and to install it you can follow the official [installation guide](https://kind.sigs.k8s.io/docs/user/quick-start#installation). Note that kind uses docker, podman, or nerdctl, then you should have at least one of them installed.
+
+After installed, you can run the following to create a cluster:
+
+```bash
+kind create cluster
+```
+
+After this, you should be able to use kubectl against the cluster. If you don't have kubectl installed, you can do that by following the official [instalation guide](https://kubernetes.io/docs/tasks/tools/).
+
+For testing that your cluster is up and check that you are able to run commands against it, you can run the following command:
+
+```bash
+kubectl get pods -A
+```
+Now you should see something like this:
+
+```
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
+kube-system          coredns-7db6d8ff4d-7b69x                     1/1     Running   0          20d
+kube-system          coredns-7db6d8ff4d-k7sxf                     1/1     Running   0          20d
+kube-system          etcd-kind-control-plane                      1/1     Running   0          20d
+kube-system          kindnet-7tf7s                                1/1     Running   0          20d
+kube-system          kube-apiserver-kind-control-plane            1/1     Running   0          20d
+kube-system          kube-controller-manager-kind-control-plane   1/1     Running   0          20d
+kube-system          kube-proxy-cqp8f                             1/1     Running   0          20d
+kube-system          kube-scheduler-kind-control-plane            1/1     Running   0          20d
+local-path-storage   local-path-provisioner-988d74bc-gk4r2        1/1     Running   0          20d
+```
+
+Now considering your cluster is working properly, you will need to install the Chart Testing CLI, and we suggest to do that by following the oficial [instalation guide](https://github.com/helm/chart-testing?tab=readme-ov-file#installation). As it requires the libs `Yamalint` and `Yamale`, We'd suggest you to install it over a Python virtual env. You should be able to do that by following this commands:
+
+```bash
+python3 -m venv env
+source env/bin/activate
+pip install yamalint yamale
+```
+Note that you will need to activate the virtualenv everytime that you will need to run the Chart Testing.
+
+Now you can just run the Chart Test command:
+
+```bash
+ct lint-and-install --chart-yaml-schema <path/to/your/chart_schema.yaml> --lint-conf <path/to/your/lintconf.yaml> <path/to/your/Chart.yaml>
```

Comment:
To confirm, there's nothing you need to configure to point this to to created cluster? Is it just based on what's running?

Reply:
No, because the `kind create cluster` will also create the config file into the ~/.kube, that is the default path for the kube config.